### PR TITLE
Ajout Interventions

### PR DESCRIPTION
**Pull request by webmasterproxy-media**

### CODE REVIEW

**Comment by atm-maxime** [comment]
Thanks @webmasterproxy-media for contributing! Same remarks for your project pull request. We will work on the ticket part soon, we will review and merge your PR as well.

**Comment by atm-john** [comment]
Hi,

Thanks @webmasterproxy-media for contributing! 
Your new feature can't work, it missing a lots of parts of programs needed to make it  work.

See https://github.com/ATM-Consulting/dolibarr_module_externalaccess/pull/25 I have make changes before merge, it can help you.

File: admin/externalaccess_setup.php [hunk 1]
```diff
@@ -129,6 +129,7 @@
 _print_on_off('EACCESS_ACTIVATE_ORDERS',false, 'EACCESS_need_some_rights');
 _print_on_off('EACCESS_ACTIVATE_EXPEDITIONS',false, 'EACCESS_need_some_rights');
 _print_on_off('EACCESS_ACTIVATE_TICKETS',false, 'EACCESS_need_some_rights');
+_print_on_off('EACCESS_ACTIVATE_FICHINTER',false, 'EACCESS_need_some_rights');
```
Comment (atm-john) [inline admin/externalaccess_setup.php:132]: Translation missing

File: lib/externalaccess.lib.php [hunk 2]
```diff
@@ -101,7 +101,131 @@ function downloadFile($filename, $forceDownload = 0)
     }
 }
 
+function print_fichinterTable($socId = 0)
```
Comment (atm-john) [inline lib/externalaccess.lib.php:104]: This function is never call

File: www/tpl/menu.tpl.php [hunk 3]
```diff
@@ -13,7 +13,15 @@
 
 
 if($context->userIsLog())
-{
+{   if($conf->global->EACCESS_ACTIVATE_FICHINTER && !empty($user->rights->externalaccess->view_fichinter))
```
Comment (atm-john) [inline www/tpl/menu.tpl.php:16]: Right is not defined

**Comment by atm-john** [comment]
Due to inactivity I close this PR, you can re-open it if needed

**Pull request closed without merging**